erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -400,11 +401,25 @@ int WebRtcConnection::deliverFeedback_(char* buf, int len) {
   return len;
 }
 
-void WebRtcConnection::onTransportData(char* buf, int len, Transport *transport) {
+void WebRtcConnection::onTransportData(std::shared_ptr<dataPacket> packet, Transport *transport) {
   if (audioSink_ == NULL && videoSink_ == NULL && fbSink_ == NULL) {
     return;
   }
 
+  packetType type = OTHER_PACKET;
```

Comment:
I think the way this is now works but it's a little awkward. I'm pretty sure using just SSRC instead of transport type should already be enough. In any case, it's not that important for this PR and could be addressed in the future